Makefile

```diff
@@ -29,7 +29,7 @@ test-install: clean
 	$(PIP) install --requirement $(TEST_REQUIREMENTS)
 
 test:
-	pytest
+	dbus-launch pytest
```

Comment:
bound to fail on OSX. You've to install the tools required first. And then debug properly